src/Jellyfin.Database/readme.md

```diff
@@ -0,0 +1,25 @@
+# How to run EFCore migrations
+
+This shall provide context on how to work with entity frameworks multi provider migration feature.
+
+Jellyfin will in the future support multiple database providers, namely SqLite as its default and the experimental postgresSQL.
```

Comment:
```suggestion
Jellyfin will support multiple database providers in the future, namely SqLite as its default and the experimental postgresSQL.
```

Not sure if we should write this here since postgres won't be in 10.11

Reply:
I mean ... future ... could also write it as "soon ™"